big_tests/tests/graphql_SUITE.erl

```diff
@@ -0,0 +1,205 @@
+-module(graphql_SUITE).
+
+-include_lib("common_test/include/ct.hrl").
+-include_lib("eunit/include/eunit.hrl").
+
+-compile([export_all, nowarn_export_all]).
+-import(distributed_helper, [mim/0, require_rpc_nodes/1, rpc/4]).
+
+suite() ->
+    require_rpc_nodes([mim]) ++ escalus:suite().
+
+all() ->
+    [{group, cowboy_handler},
+     {group, admin_handler},
+     {group, user_handler}].
+
+groups() ->
+    [{cowboy_handler, [], [can_connect_to_admin,
+                           can_connect_to_user
+                          ]},
+     {user_handler, [], [wrong_creds_cannot_access_protected_types,
```

Comment:
I think some cases (or even all) can be run in parallel.
except this means you need to use fresh users :) (i.e. fresh_story)